vspace/src/main/webapp/WEB-INF/views/staff/images/edit.html

```diff
@@ -1,29 +1,25 @@
 <html layout:decorate="~{layouts/main_staff}">
-<head>
-<link th:href="@{/resources/bootstrap-4.1.2/login.css}" rel="stylesheet" />
-</head>
 <body>
-    <div layout:fragment="content">
+    <div class="main-content" layout:fragment="content">
         <h1>Edit Image: [[${imageForm.fileName}]]</h1>
-        <form th:action="@{'/staff/image/'+${imageId}+'/edit'}"
-            method="POST" name="f">
-            <input type="hidden" name="${_csrf.parameterName}"
-                value="${_csrf.token}" />
+        
+        <form method="POST"
+        	th:action='@{|/staff/image/${imageId}/edit?${_csrf.parameterName}=${_csrf.token}|}' th:object="${imageForm}" 
```

Comment:
why did you move the csrf token here? it should stay in the body.

Reply:
As discussed over zoom, I have removed this and now the form is working.